Source/WTF/wtf/HashMap.h

```diff
@@ -95,7 +95,7 @@ class HashMap final {
         for (const auto& keyValuePair : initializerList)
             add(keyValuePair.key, keyValuePair.value);
     }
-    
+
```

Comment:
All the changes to this file are just trimming white space. I can't think of a compelling reason to do that and such changes should not be combined with substantive changes in the same pull request.